htdocs/core/tpl/objectline_create.tpl.php

```diff
@@ -675,13 +676,26 @@ function checkFreeLine(e, npRate)
 	}
 	?>
 
+<?php if ($this->table_element_line != 'commande_fournisseurdet' || $this->table_element_line == 'facture_fourn_det') { ?>
```

Comment:
This test looks strange
!=   ||   ==
Can you check

Reply:
yep  ;) i removed the second one. thx